pkg/types/operation.go

```diff
@@ -0,0 +1,89 @@
+/*
+ *    Copyright 2018 The Service Manager Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/Peripli/service-manager/pkg/util"
+)
+
+// OperationCategory is the type of an operation
+type OperationCategory string
+
+// OperationState is the state of an operation
+type OperationState string
+
+const (
+	// CREATE represents an operation type for creating a resource
+	CREATE OperationCategory = "CREATE"
+
+	// UPDATE represents an operation type for updating a resource
+	UPDATE OperationCategory = "UPDATE"
+
+	// DELETE represents an operation type for deleting a resource
+	DELETE OperationCategory = "DELETE"
+
+	// SUCCEEDED represents the state of an operation after successful execution
```

Comment:
can you split this in two groups as it would be easier to read? 
```golang
type .. string
const (...)

type ... string
const (....)
```

Reply:
done